templates/web/src/sdk.ts.twig

```diff
@@ -9,6 +9,55 @@ type Headers = {
     [key: string]: string;
 }
 
+type RealtimeResponse = {
+    type: "error"|"event"|"connected"|"response";
+    data: RealtimeResponseAuthenticated|RealtimeResponseConnected|RealtimeResponseError|RealtimeResponseEvent;
+}
+
+type RealtimeRequest = {
+    type: "authentication";
+    data: RealtimeRequestAuthenticate;
+}
+
+type RealtimeResponseEvent = {
+    event: string;
+    channels: string[];
+    timestamp: number;
+    payload: unknown;
+}
+
+type RealtimeResponseError = {
+    code: number;
+    message: string;
+}
+
+type RealtimeResponseConnected = {
+    channels: string[];
+    user?: object;
+}
+
+type RealtimeResponseAuthenticated = {
+    to: string;
+    success: boolean;
+    user: object;
+}
+
+type RealtimeRequestAuthenticate = {
+    session: string;
+}
+
+type RealtimeObject = {
```

Comment:
Why not just `Realtime` instead of `RealtimeObject`? will be more in sync with other SDKs as well?

Reply:
Good idea 👍🏻 

Will do.